react/Button/Button.less

```diff
@@ -111,3 +111,47 @@
 .fullWidth {
   width: 100%;
 }
+
+.group {
+  display: flex;
+  width: 100%;
+
+  .root {
+    overflow: visible;
+
+    & + .root {
+      margin-left: @grid-gutter-width / 2;
+    }
+
+    & + .root_isTransparent {
+      margin-left: 0;
+    }
+  }
+
+  .root_isTransparent {
+    padding-left: @grid-gutter-width;
+    padding-right: @grid-gutter-width;
+  }
+  @media only screen and (max-width: 960px) {
+    .root {
+      flex: 33.33% 0;
+    }
+
+    .root_isTransparent {
+      flex: 20% 0;
+    }
+  }
+  @media only screen and (max-width: 500px) {
```

Comment:
seems very specific to me, because what if I want to implement them in a narrow container for example. will be weird